src/app/mod.rs

```diff
@@ -26,8 +26,17 @@ fn print_options(config_filename: &str, tokens: &[String], last_token: &str) ->
     let options_finder = options_finder::OptionsFinder::new(result);
     let opts = options_finder.options(last_token)?;
 
+    let print_descriptions = std::env::var("TABRY_PRINT_DESCRIPTIONS").is_ok();
+
     for opt in &opts.options {
-        println!("{}", opt);
+        if print_descriptions {
+            match opt.desc.clone() {
```

Comment:
You shouldn't need to clone, you should be able to use `match opt.as_ref() {` . That converts an `&Option<T>` to an `Option<&T>`, so will prevent moving opt.desc.